File: 1100/1463A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        long long a, b, c;
        cin >> a >> b >> c;

        long long sum = a + b + c;

        if (sum % 9 == 0 && a >= sum / 9 && b >= sum / 9 && c >= sum / 9)
            cout << "YES\n";
        else
            cout << "NO\n";
    }
    
    return 0;
}

File: 1100/1463A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            long a = sc.nextLong();
            long b = sc.nextLong();
            long c = sc.nextLong();
            
            long sum = a+b+c;
            
            if(sum%9 == 0 && a >= sum/9 && b >= sum/9 && c >= sum/9)
                System.out.println("YES");
            else
                System.out.println("NO");
        }
    }
}